package/MDAnalysis/analysis/encore/clustering/ClusteringMethod.py

```diff
@@ -215,7 +214,7 @@ def __call__(self, distance_matrix):
             Returns
             -------
             numpy.array
-                list of cluster indices
+                array of centroid frame indices
```

Comment:
Could you please add the shape of the array it returns -- right now it sounds like it's returning an array of shape (n_clusters) where the elements are the centroid frame indices, whereas actually it's an array of (n_elements) where clusters[i] returns the centroid frame, of the cluster for element i

Reply:
Sure 👍